src/sentry/constants.py

```diff
@@ -264,7 +264,8 @@ def get_all_languages() -> list[str]:
     "sentry.rules.conditions.first_seen_event.FirstSeenEventCondition",
     "sentry.rules.conditions.regression_event.RegressionEventCondition",
     "sentry.rules.conditions.reappeared_event.ReappearedEventCondition",
-    "sentry.rules.conditions.high_priority_issue.HighPriorityIssueCondition",
+    "sentry.rules.conditions.new_high_priority_issue.NewHighPriorityIssueCondition",
+    "sentry.rules.conditions.existing_high_priority_issue.ExistingHighPriorityIssueCondition",
```

Comment:
Are there rules that exist with `HighPriorityIssueCondition`? If so, we need to keep that condition in place until we migrate them

Reply:
Yep, I'll go with the 'clean it up after' route just to be safe